ironfish-cli/src/commands/wallet/notes/combine.ts

```diff
@@ -76,6 +76,11 @@ export class CombineNotesCommand extends IronfishCommand {
       default: false,
       description: 'Force run the benchmark to measure the time to combine 1 note',
     }),
+    rawTransaction: Flags.boolean({
+      default: false,
+      description:
+        'Return raw transaction. Use it to create a transaction but not post to the network',
+    }),
   }
```

Comment:
What is the use case for this, can you include in PR description?

Reply:
Updated